src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/Microsoft.CodeAnalysis.LanguageServices.vsmanproj

```diff
@@ -13,6 +13,10 @@
   <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
   <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.targets" />
 
+  <ItemGroup>
+    <ProjectReference Include="..\MicrosoftVisualStudioInteractiveWindow\Microsoft.VisualStudio.InteractiveWindow.vsmanproj" />
+  </ItemGroup>
+
```

Comment:
Microsoft.CodeAnalysis.LanguageServices.vsmanproj is an umbrella for all Roslyn stuff, including interactive language services.